src/get_input.py

```diff
@@ -193,8 +193,7 @@ def write_profile(ctx, slug: str):
     # try to find device and profile JSON files
     if isdir(profile_dir):
         device, profile = load_profile(profile_dir)
-    # else try to download the profile from API
-    if not (device and profile):
+    else:
```

Comment:
I'm not convinced this is the correct solution.  I think a more correct fix might be declaring `device, profile = None` above the isdir check, and changing it back to an `if device is None or profile is None:`.  This would also potentially cover if only 1 of 2 files was present, in which case in the isdir if clause, we should add an additional message if one is None, otherwise it would be unclear that it tried a custom profile directory and failed, as the only message would be the API failed.

Reply:
Ha ha, I had something similar, but couldn't trigger that on quick testing, agreed, that's more correct, and the `print` is better as well. Made a minor fix and committed.